src/main/java/com/bravo/user/dao/model/Profile.java

```diff
@@ -0,0 +1,41 @@
+package com.bravo.user.dao.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.bravo.user.model.dto.ProfileSaveDto;
+
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "profile")
+public class Profile {
+
+  @Id
+  @Column(name = "id")
+  private String id;
+
+  @Column(name = "image_ref", nullable = false)
+  private String imageRef;
+
+  @Column(name = "updated", nullable = false)
+  private LocalDateTime updated;
+
+  public Profile() {
+    super();
+    this.id = UUID.randomUUID().toString();
+    this.updated = LocalDateTime.now();
```

Comment:
Having the update time be when the object is created may not match the actual update time.

Reply:
This is true - I'll admit I went with the design in the other VO due to time. I'd expect the update time to either be populated immediately before the create/update action or via a trigger in the db.